src/server.js

```diff
@@ -208,6 +208,7 @@ function init(socket, client) {
 		socket.on(
 			"open",
 			function(data) {
+				client.lastActiveSocket = socket;
```

Comment:
It isn't quite `last connected` either, it's the last socket that sent `open` (i.e. the last client that clicked on a channel).